wireapp/wire-android-reloaded
# feat: (AR-1726) Long Tap on Images doesn't bring up message option menu

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1726" title="AR-1726" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />AR-1726</a>  Long Tap on Images doesn't bring up message option menu (i.e. delete, copy...)
  </td></table>
  <br />
 






----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues
Long press on images did nothing, well actually it behaved like normal clicked but that's not expected
### Causes (Optional)

_Briefly describe the causes behind the issues. This could be helpful to understand the adopted solutions behind some nasty bugs or complex issues._

### Solutions

Long press should open context menu

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

1. login to wire app
2. open any conversation
3. send an image through the image button
4. Long press on the sent image
expected: show context menu

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
https://wearezeta.atlassian.net/browse/AR-1726
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
